src/main/java/org/jabref/gui/theme/ThemeManager.java

```diff
@@ -213,7 +215,7 @@ public void updateFontStyle(Scene scene) {
         if (appearancePreferences.shouldOverrideDefaultFontSize()) {
             scene.getRoot().setStyle("-fx-font-size: " + appearancePreferences.getMainFontSize() + "pt;");
         } else {
-            scene.getRoot().setStyle("");
+            scene.getRoot().setStyle("-fx-font-size: " + JabRefPreferences.getInstance().getDefaults().get(JabRefPreferences.MAIN_FONT_SIZE) + "pt;");
```

Comment:
This is ugly. Add a method in the appearence preferences, call it something like getDefaultFontSize() and pass the default value to it.

Reply:
@Siedlerchr Agreed, it is not an ideal solution. 
I have added the `getDefaultFontSize()` method to the `AppearancePreferences` class. The default value is passed to the class in the constructor. The ThemeManager now uses the getter method to set the default font size :)

Reply:
Thanks this looks much better now!